### PR TITLE
To make it work with clean-webpack-plugin v3

### PR DESCRIPTION
**Pull request by veeramarni**

### CODE REVIEW

File: packages/zen/src/plugins/WebpackPlugin.ts [hunk 1]
```diff
@@ -71,8 +71,8 @@ const createPlugins = (builder: Builder, zen: Zen) => {
   }
 
   if (builder.require.probe('clean-webpack-plugin')) {
-    const CleanWebpackPlugin = builder.require('clean-webpack-plugin');
-    plugins = plugins.concat(new CleanWebpackPlugin(builder.buildDir));
+    const { CleanWebpackPlugin } = builder.require('clean-webpack-plugin');
+    plugins = plugins.concat(new CleanWebpackPlugin());
```
Comment (larixer) [inline packages/zen/src/plugins/WebpackPlugin.ts:75]: @veeramarni Thanks, why don't you pass `builder.buildDir` into `CleanWebpackPlugin` constructor, as it was done before? Can we support both old and new `clean-webpack-plugin` via code like this:
```
    let CleanWebpackPlugin = builder.require('clean-webpack-plugin');
    CleanWebpackPlugin = CleanWebpackPlugin.CleanWebpackPlugin || CleanWebpackPlugin;
    plugins = plugins.concat(new CleanWebpackPlugin(builder.buildDir));
```

Comment (veeramarni) [reply packages/zen/src/plugins/WebpackPlugin.ts:75]: Will make that change.

**Comment by veeramarni** [comment]
@larixer `builder.buildDir` is `undefined` even if we pass it. It is `undefined` in that file and `CleanWebpackPlugin` needs an object in [v2](https://github.com/johnagan/clean-webpack-plugin#options-and-defaults-optional) and [v3](https://github.com/johnagan/clean-webpack-plugin#options-and-defaults-optional), not sure why it is passed so removed.

**Comment by larixer** [comment]
@veeramarni It's undefined in your case, but it can be defined, why not pass it?

**Comment by veeramarni** [comment]
@larixer I can keep it, but does `CleanWebpackPlugin` take the `string` value instead of `object`? I cannot find the documentation for it. But if you insist I can keep it.

**Comment by larixer** [comment]
@veeramarni `builder.buildDir` is a string, it can have `undefined` value

**Comment by veeramarni** [comment]
I mean from its documentation it takes `object`, not `string` like `CleanWebpackPlugin<{object}>`. But I can keep the old way.

**Comment by veeramarni** [comment]
@larixer pushed with `buildDir` in the constructor.

**Comment by larixer** [comment]
@veeramarni I see, than for older versions of `clean-webpack-plugin` `builder.buildDir` string should be passed, and for new versions the correct object should be passed.

**Comment by larixer** [comment]
I will be fine, if you just pass `builder.buildDir` to old versions and pass no params for new versions

**Comment by larixer** [comment]
```ts
    const CleanWebpackPlugin = builder.require('clean-webpack-plugin');
    plugins = plugins.concat(CleanWebpackPlugin.CleanWebpackPlugin ? new CleanWebpackPlugin.CleanWebpackPlugin() : new CleanWebpackPlugin(builder.buildDir));
 ```

**Comment by larixer** [comment]
Thanks!

**Pull request merged**